docs/en/observability/cloud-monitoring/aws/monitor-aws-cloudtrail-firehose.asciidoc

```diff
@@ -0,0 +1,207 @@
+[[monitor-aws-cloudtrail-firehose]]
+= Monitor CloudTrail logs
+
+++++
+<titleabbrev>Monitor CloudTrail logs</titleabbrev>
+++++
+
+In this section, you'll learn how to monitor and analyze the CloudTrail logs you send to Elastic with Amazon Data Firehose. You will go through the following steps:
+
+- Install AWS integration in {kib}
+- Export Cloudtrail events to CloudWatch
+- Set up a Firehose delivery stream
+- Set up a subscription filter to route Cloudtrail events to a delivery stream
+- Visualize your CloudTrail logs in {kib}
+
+
+[discrete]
+[[firehose-cloudtrail-prerequisites]]
+== Before you begin
+
+We assume that you already have:
+
+- An AWS account with permissions to pull the necessary data from AWS.
+- A deployment using our hosted {ess} on {ess-trial}[{ecloud}]. The deployment includes an {es} cluster for storing and searching your data, and {kib} for visualizing and managing your data. AWS Data Firehose works with Elastic Stack version 7.17 or greater, running on Elastic Cloud only.
+
+IMPORTANT: Make sure the deployment is on AWS, because the Amazon Data Firehose delivery stream connects specifically to an endpoint that needs to be on AWS.
+
+[discrete]
+[[firehose-cloudtrail-step-one]]
+== Step 1: Install AWS integration in {kib}
+
+. Install AWS integrations to load index templates, ingest pipelines, and dashboards into {kib}. In {kib}, navigate to *Management* > *Integrations* and find the AWS Integration by browsing the catalog.
+
+. Navigate to the *Settings* tab and click *Install AWS assets*. Confirm by clicking *Install AWS* in the popup.
+
+. Install AWS Firehose integration assets in Kibana. 
+
+NOTE: Amazon Data Firehose integration is currently in beta. Make sure to enable *Display beta integrations*.
+
+[discrete]
+[[firehose-cloudtrail-step-two]]
+== Step 2: Export Cloudtrail events to CloudWatch
+
+image::firehose-cloudtrail-cloudwatch.png[Cloudtrail to CloudWatch]
+
+To export CloudTrail logs to CloudWatch, you must set up a *trail* through the following steps:
+
+. Go to the https://console.aws.amazon.com/[AWS console] and navigate to CloudTrail.  
+
+. Click *Create trail* and configure the general details on the *Choose trail attributes* panel, like for example:
++
+* Trail name
+* Storage location
+
+By default, CloudTrail exports data to an S3 bucket. It isn’t possible to opt-out from S3.
```

Comment:
Because of this paragraph, the ordered list starts over at 1. I don't think that's what we want? We can fix the ordering by adding a +

```suggestion
+
By default, CloudTrail exports data to an S3 bucket. It isn’t possible to opt-out from S3.
```